src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -750,6 +750,8 @@ def runK8(image, branchName, config, axis, steps=config.steps) {
     def requests = image.requests ?: getConfigVal(config, ['kubernetes', 'requests'], "{memory: 8Gi, cpu: 4000m}")
     def annotations = image.annotations ?: getConfigVal(config, ['kubernetes', 'annotations'], [], false)
     def caps_add = image.caps_add ?: getConfigVal(config, ['kubernetes', 'caps_add'], "[]")
+    def service_account =  getConfigVal(config, ['kubernetes', 'serviceAccount'], "default")
+    def namespace =  getConfigVal(config, ['kubernetes', 'namespace'], "default")
```

Comment:
Namespace is also useful in function `build_docker_on_k8`. Not sure about service account.

Reply:
> Namespace is also useful in function `build_docker_on_k8`. Not sure about service account.

no, `docker build` with mounted `/var/run/docker.sock` is executed on the host directly, not under K8s control (accounting.